packages/task/src/browser/task-frontend-contribution.ts

```diff
@@ -98,6 +107,14 @@ export class TaskFrontendContribution implements CommandContribution, MenuContri
                 execute: () => this.quickOpenTask.attach()
             }
         );
+        registry.registerCommand(
+            TaskCommands.TASK_RUN_LAST,
+            {
+                isVisible: () => !!this.taskService.getLastTask(),
```

Comment:
Just trying to understand: what does isVisible influence?  For example, if we have a menu entry for a command and isVisible is false, will the menu entry not appear?

If there is not recently last ran task, I would just expect the item to be present, but greyed out (which I suppose is controlled by isEnabled).

Reply:
To be honest, the `isVisible` part seems broken since in electron it has no effect with their menus. `isEnabled` however provides the behavior you like 'greyed out' and hide from the `command palette`. This discussion made me think and I decided it might also be useful to add a menu item for `Run Last Task` :)

<div align='center'>

<img width="545" alt="screen shot 2019-03-01 at 5 26 49 pm" src="https://user-images.githubusercontent.com/40359487/53670038-38838a00-3c47-11e9-967b-0cdc22a10d1c.png">

</div>